index.html

```diff
@@ -32,11 +41,170 @@ <h1 class="bio-header">I'm Alexandra. Glad to see you!</h1>
+                    <li class="card-prop"><img src="img/Counter.png"></li>
+                    <li  class="card-prop">Canopy</li>
+        </ul>
+        <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+         <ul class="card-languages">
+            <li class="card-language">HTML</li>
+            <li class="card-language">CSS</li>
+            <li class="card-language">Javascript</li>
+            
+         </ul>
+         <button class="card-btn">See Project</button>
+    </div>
+
+
+    <div class="card">
+        <div class="card-img">
+            <img src="img/work-3.png">
+        </div>
+        <h2>Tonic</h2>
+        <ul class="card-props">
+            <li  class="card-prop">Canopy</li>
+                    <li class="card-prop"><img src="img/Counter.png"></li>
+                    <li class="card-prop">Back End Dev</li>
+                    <li class="card-prop"><img src="img/Counter.png"></li>
+                    <li  class="card-prop">Canopy</li>
+        </ul>
+        <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+         <ul class="card-languages">
+            <li class="card-language">HTML</li>
+            <li class="card-language">CSS</li>
+            <li class="card-language">Javascript</li>
+            
+         </ul>
+         <button class="card-btn">See Project</button>
+    </div>
+
+
+    <div class="card">
+        <div class="card-img">
+            <img src="img/work-3.png">
+        </div>
+        <h2>Tonic</h2>
+        <ul class="card-props">
+            <li  class="card-prop">Canopy</li>
+                    <li class="card-prop"><img src="img/Counter.png"></li>
+                    <li class="card-prop">Back End Dev</li>
+                    <li class="card-prop"><img src="img/Counter.png"></li>
+                    <li  class="card-prop">Canopy</li>
+        </ul>
+        <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+         <ul class="card-languages">
+            <li class="card-language">HTML</li>
+            <li class="card-language">CSS</li>
+            <li class="card-language">Javascript</li>
+            
+         </ul>
+         <button class="card-btn">See Project</button>
+    </div>
+</section>
+
+<section class="about-section">
+    <div class="content">
+        <div class="about-1">
     
+            <h2 class="about-heading">About Myself</h2>
+                    <p class="about-text">
+                        Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                    </p>
+                    <div class="connect">
+                        <hx class="connect-text">lets connect</hx>
```

Comment:
- [ ] 🙏Please replace the existing code👩‍💻 with the following line, as suggested by the previous reviewer🙄:
```
<h2 class="connect-text">Let's connect</h2>
```
The previous reviewer🙄 suggested using `hx`, which means you can use any of the following heading tags: `h2, h3, h4, or h5`. This is considered the best👍 practice, especially when used as a subheading.

Reply:
Thank you very much for reviewing my project